scripts/release.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+const {argv, env, exit} = require('process');
+const {execSync} = require('child_process');
+
+const {loadEnv} = require('./common');
+
+const version = argv[2];
+
+if (!version) {
+    console.error("Usage: release.js <version>")
+    exit(1);
+}
+
+const command = `towncrier build --yes --version=${version}`;
```

Comment:
Might be worth a comment that this will `git rm` the newsfragments and produce a new commit (with the NEWS or whatever file).